src/main/java/Entities/Property.java

```diff
@@ -3,57 +3,83 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.util.ArrayList;
+import java.util.Objects;
 import java.util.Scanner;
 
 public class Property extends Cell {
-    private Player ownedBy;
-    private int houses;
-    private int rent;
     private String name;
     private String colour;
+    private int cost;
+    private int rent;
+    private int rent1H;
+    private int rent2H;
+    private int rent3H;
+    private int rent4H;
+    private int rentHotel;
+    private Player ownedBy;
     private int mortgageValue;
     private int houseCost;
-    private int cost;
+    private int houses;
+    private boolean mortgaged;
 
-    public Property (String name, String colour, int cost, int rent, int mortgageValue, int houseCost) {
+    public Property (String name, String colour, int cost, int houseCost, int rent,
+                     int rent1H, int rent2H, int rent3H, int rent4H, int rentHotel,
+                     int mortgageValue) {
+        this.name = name;
+        this.houseCost = houseCost;
+        this.colour = colour;
+        this.cost = cost;
+        this.rent = rent;
+        this.rent1H = rent1H;
+        this.rent2H = rent2H;
+        this.rent3H = rent3H;
+        this.rent4H = rent4H;
+        this.rentHotel = rentHotel;
         this.ownedBy = null;
+        this.mortgageValue = mortgageValue;
         this.houses = 0;
-        this.cost = cost;
```

Comment:
set mortgaged to false

Reply:
removing this constructor in favor of the second as a general with Player owner = "none", int houses = 0, and boolean mortgaged "false" for default